internal/csv/reader.go

```diff
@@ -0,0 +1,412 @@
+// Copyright 2011 The Go Authors. All rights reserved.
```

Comment:
```suggestion
// Note: this package is a fork of the Go standard library's encoding/csv
// package that adds the capability to find the current line number.
// If/when https://github.com/golang/go/issues/44221 is accepted and implemented,
// this can be removed.

// Copyright 2011 The Go Authors. All rights reserved.
```

Reply:
This is marked as resolved, but doesn't seem to be resolved to me. We should include the LICENSE file, and I still think it would be worth adding a comment explaining why we're vendoring this package, as suggested above.